src/Microsoft.AspNet.Mvc.Xml/DelegatingEnumerable.cs

```diff
@@ -0,0 +1,73 @@
+﻿// Copyright (c) Microsoft Open Technologies, Inc. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+using System.Collections;
+using System.Collections.Generic;
+using System.Linq;
+
+namespace Microsoft.AspNet.Mvc.Xml
+{
+    /// <summary>
+    /// Helper class to serialize <see cref="IEnumerable{T}"/> types by delegating them through a concrete 
```

Comment:
leave our terms like `Helper class` when writing docs. It's not meaningful. Just say `Serializes ...`

Get right to the point :laughing: